manual/dotnet4.7.2-deu/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,30 @@
+﻿$ErrorActionPreference = 'Stop'
+
+// For release numbers see https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed#to-find-net-framework-versions-by-viewing-the-registry-net-framework-45-and-later
+
+if ($null -eq $props -or $props.Release -lt 461808) {
+    throw ".NET Framework 4.7.2 is not installed."
+}
+
+if ($props.Release -gt 461814) {
+    throw "There's a newer version of .NET Framework than 4.7.2 installed."
+}
+
+$packageArgs = @{
+    packageName   = $env:ChocolateyPackageName
+    softwareName  = 'Microsoft .NET Framework  (DEU)*'
```

Comment:
Seems to be a space too much after `Framework` and before `(DEU)` ?

Same as for `4.6.2-deu` [here](https://github.com/bbtsoftware/chocolatey-packages/blob/master/manual/dotnet4.6.2-deu/tools/chocolateyInstall.ps1).